lib/block/block.js

```diff
@@ -262,15 +262,9 @@ Block.prototype.getMerkleRoot = function getMerkleRoot() {
  * @returns {Boolean} - If the merkle roots match
  */
 Block.prototype.validMerkleRoot = function validMerkleRoot() {
```

Comment:
I think this method will not work with zendoo

Reply:
Do you think we should update it? What exactly wouldn't work?

Reply:
Remove it